packages/x-components/src/components/panels/base-id-toggle-panel-button.vue

```diff
@@ -4,6 +4,8 @@
     :events="events"
     class="x-button x-base-id-toggle-panel-button"
     data-test="base-id-toggle-button"
+    :aria-pressed="isPanelOpen"
+    aria-labelledby="base-id-toggle-button"
```

Comment:
I put a similar comment on other file, should we use here the panel ID to make sure we don't generate any conflict when using more than one panel at the same time?
```suggestion
    :aria-labelledby="`base-id-toggle-button-${panelId}`"
```